common/test/views/support/cleaner/AtomCleanerTest.scala

```diff
@@ -93,6 +96,36 @@ class AtomCleanerTest extends FlatSpec
     doc.getElementsByClass("youtube-media-atom__bottom-bar__duration").html() should be("0:36")
   }
 
+  "Youtube template" should "use active asset" in {
+    val atom = youTubeAtom.get.media.head.copy(
+      activeVersion = Some(2L),
+      assets = Seq(
+        youTubeAsset,
+        youTubeAsset.copy(id = "gyVuRflcEKM", version = 2),
+        youTubeAsset.copy(id = "QRplDNMsS4U", version = 3)
+      )
+    )
+
+    val html = views.html.fragments.atoms.media(media = atom, displayCaption = false, mediaWrapper = None)(TestRequest())
+    val doc = Jsoup.parse(html.toString())
+
+    doc.getElementsByClass("youtube-media-atom__iframe").attr("id") should be("youtube-gyVuRflcEKM")
```

Comment:
You could abstract this code you reuse twice.
Ex:
```
def atomIdInHtml(atom: MediaAtom) {
    val html = views.html.fragments.atoms.media(media = atom, displayCaption = false, mediaWrapper = None)(TestRequest())
    doc = Jsoup.parse(html.toString())
    doc.getElementsByClass("youtube-media-atom__iframe")
}
...
atomIdInHtml(atom) should be("youtube-gyVuRflcEKM") 
```
`atomIdInHtml` might not be the best name though